cmd/helm/root.go

```diff
@@ -153,15 +153,26 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 	flags.ParseErrorsWhitelist.UnknownFlags = true
 	flags.Parse(args)
 
-	registryClient, err := registry.NewClient(
-		registry.ClientOptDebug(settings.Debug),
-		registry.ClientOptWriter(out),
-		registry.ClientOptCredentialsFile(settings.RegistryConfig),
-	)
-	if err != nil {
-		return nil, err
+	cmd.PersistentPreRunE = func(cmd *cobra.Command, args []string) error {
+		registryClient, err := registry.NewClient(
+			registry.ClientOptDebug(settings.Debug),
+			registry.ClientOptWriter(out),
+			registry.ClientOptCredentialsFile(settings.RegistryConfig),
+			registry.ClientOptPlainHTTP(registry.PlainHTTPOpt),
+			registry.ClientOptInsecureSkipVerifyTLS(registry.InsecureOpt),
```

Comment:
@bacongobbler - specifically I think this option needs to be carried over if is not already